html/lesson10.js

```diff
@@ -0,0 +1,195 @@
+/*
+ Bai 1: In một bảng nhân của một số bất kỳ với các số từ 1 tới 10 và hiển thị kết quả
+*/
+
+const printTableMultiple = num => {
+
+    //Doi voi truong hop num là undefined hoăc null
+    if (num != 0 && !num) {
+        console.log('Xin hay nhập 1 giá trị khác null và undefined')
+        return
```

Comment:
Có thể viết ngắn thành
```suggestion
        return console.log('Xin hay nhập 1 giá trị khác null và undefined')
```

Nhưng, không phải `null` và `undefined` thì chắc gì đã là `Number`, đã check thì check cho đến nơi đến chốn